OpenRA.Mods.Common/LoadScreens/BlankLoadScreen.cs

```diff
@@ -21,16 +21,21 @@ namespace OpenRA.Mods.Common.LoadScreens
 {
 	public class BlankLoadScreen : ILoadScreen
 	{
-		public LaunchArguments Launch;
```

Comment:
Then this becomes `public LaunchArguments Launch { get; private set; }` and the rest of the changes in the file can be discarded.